### PR TITLE
Feat: re order deposit tokens

### LINKED ISSUE #586: re-order deposit tokens
**todo:**

Re-order deposit tokens for Trend and Momentum strategies so that WBTC, WETH, and USDC will be first.

Re-order deposit tokens for Steady BTC - WBTC, WETH, and USDC will be first.

Re-order deposit tokens for Steady ETH - WETH, WBTC, and USDC will be first.

Image

### PR DESCRIPTION
**Pull request by joshuanatanielnm**
Fixes #586 

Reference: https://stackoverflow.com/questions/35538509/sort-an-array-of-objects-based-on-another-array-of-ids

## Description

re order deposit tokens 

## Changes

List any technical changes.

- [x] [feat: update tokenConfig logic](https://github.com/strangelove-ventures/sommelier/commit/3e2baa299e32818d290538ba8d37bf08414ef5ee)
- [x] [feat: update strategies depositTokens list](https://github.com/strangelove-ventures/sommelier/commit/836569776ef2a8508401bf5aed83e38d52c454df)
- [x] [fix: typescript error](https://github.com/strangelove-ventures/sommelier/commit/1d7565cd38901ecca1fa58c5d20664933a186258)

## Screenshots (if appropriate):

image
image

### CODE REVIEW

**Review by codingki — APPROVED** [review]
You are referencing https://stackoverflow.com/questions/35538509/sort-an-array-of-objects-based-on-another-array-of-ids but there's no "sorting" logic😂 it's hardcoded from what I've seen. But still looks good to me👍

File: src/data/tokenConfig.ts [hunk 1]
```diff
@@ -100,22 +101,15 @@ export const depositAssetTokenList = [
...
-
 export const depositAssetTokenConfig: Token[] = tokenConfig.filter(
   (token) => depositAssetTokenList.includes(token.symbol)
 )
 
 export function getTokenConfig(tokenList: string[]) {
-  return tokenConfig.filter((token) =>
-    tokenList.includes(token.symbol)
+  return tokenList.map((list) =>
+    tokenConfig.find((token) => token.symbol === list)
```
Comment (codingki) [inline src/data/tokenConfig.ts:113]: non blocking, but this is the same logic

Comment (joshuanatanielnm) [reply src/data/tokenConfig.ts:113]: Is not the same, because with this, the order is based on the passed array not from tokenConfig

Comment (codingki) [reply src/data/tokenConfig.ts:113]: I think it's the same 
prev: filter the `tokenConfig`by "is the `token` in the `tokenList` array. 
yours: map through the `tokenList` array and return the `tokenConfig` that has the `symbol` same as `tokenList` key 
You just change how it creates the output array.

Comment (codingki) [reply src/data/tokenConfig.ts:113]: but it's the same output so no problem then

Comment (codingki) [reply src/data/tokenConfig.ts:113]: ah I see it now, we need the sorting and you ordered it from `tokenList` ok

**Pull request merged**